app/views/home/home.html.erb

```diff
@@ -4,4 +4,7 @@
 <%= form_tag parse_imports_path, multipart: true do %>
   <%= file_field_tag :file, accept: ".xls,.xlsx" %>
   <%= submit_tag "Import" %>
-<% end %> 
\ No newline at end of file
+<% end %> 
+
```

Comment:
We will have to separate out the import and the take assessment into different views at some point.